pytorch_lightning/utilities/nan.py

```diff
@@ -0,0 +1,39 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
could be named `non_finite` to generalize NaN/Inf handling

Reply:
rather `feasible_values`? 
the `nan` can be simply swapped with just a values e.g. `np.nan`

Reply:
@ananthsub Are you making this change?  I'd say `finite_checks.py` sounds self-explanatory

Reply:
renamed to `finite_checks`